commands/hubdb/delete.ts

```diff
@@ -4,34 +4,61 @@ const { logError } = require('../../lib/errorHandlers/index');
 const { deleteTable } = require('@hubspot/local-dev-lib/api/hubdb');
 const { loadAndValidateOptions } = require('../../lib/validation');
 const { trackCommandUsage } = require('../../lib/usageTracking');
-
 const {
   addConfigOptions,
   addAccountOptions,
   addUseEnvironmentOptions,
 } = require('../../lib/commonOpts');
+const {
+  selectHubDBTablePrompt,
+} = require('../../lib/prompts/selectHubDBTablePrompt');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
 const { i18n } = require('../../lib/lang');
 
 const i18nKey = 'commands.hubdb.subcommands.delete';
 
-exports.command = 'delete <tableId>';
+exports.command = 'delete [table-id]';
 exports.describe = i18n(`${i18nKey}.describe`);
 
 exports.handler = async options => {
-  const { tableId, derivedAccountId } = options;
+  const { force, derivedAccountId } = options;
 
   await loadAndValidateOptions(options);
 
   trackCommandUsage('hubdb-delete', null, derivedAccountId);
 
   try {
+    const { tableId } =
+      'tableId' in options
+        ? options
+        : await selectHubDBTablePrompt({
+            accountId: derivedAccountId,
+            options,
+            isDeleteCommand: true,
+          });
+
+    if (!force) {
+      const { shouldDeleteTable } = await promptUser({
+        name: 'shouldDeleteTable',
+        type: 'confirm',
+        message: i18n(`${i18nKey}.shouldDeleteTable`, { tableId }),
+      });
+      process.stdin.resume();
```

Comment:
Is this required for the prompt to function?

Reply:
Nope, going to remove.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1267/commits/78e18349a588f00f0104bb1b53ef5284563207cb